### ISSUE TITLE
Maybe include scene/act shifts?

### ISSUE THREAD

**Issue by oakfang** [BobAdamsEE/SouthParkData#1]
What do you think about keeping the locations and act-end rows in?

Basically, I'm thinking someone might be able to train some AI on it to show the different conversation flows in the series, and an "endmarker" would be very helpful in this regard.

**Comment by BobAdamsEE** [comment]
I agree that it would add more information but for NLP purposes it would get in the way.  I'd like to have a second set which leaves the information in but I don't have the time to clean up all of the scene information.

There are issues with the source data that I think would make it difficult to process; Examples include doubled up stage notes that could mistakenly suggest an empty scene or flashback sequences in a single block which would be missed (See [here](http://southpark.wikia.com/wiki/Cartman_Gets_an_Anal_Probe/Script))
